core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -709,7 +727,20 @@ public boolean isSensitive() {
    * @return true if property is sensitive
    */
   public static boolean isSensitive(String key) {
```

Comment:
I had thought we had a similar method for Experimental and Deprecated with arbitrary keys also, which would check based on the prefix... but perhaps not. I don't see it, but it could be useful for the shell config command output.